CI workflow and ./m quickinstall failing to write _version.py

Getting following error with `./m quickinstall`

```
py39: packaging backend failed (code=1), with AssertionError: write_to=PosixPath('/home/moint03/test/moin/src/moin/_version.py')
/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools/_distutils/dist.py:265: UserWarning: Unknown distribution option: 'message_extractors'
  warnings.warn(msg)
Traceback (most recent call last):
  File "/home/moint03/test/moin-venv-python3/lib/python3.9/site-packages/pyproject_api/_backend.py", line 90, in run
    outcome = backend_proxy(parsed_message["cmd"], **parsed_message["kwargs"])
  File "/home/moint03/test/moin-venv-python3/lib/python3.9/site-packages/pyproject_api/_backend.py", line 32, in __call__
    return getattr(on_object, name)(*args, **kwargs)
  File "/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools/build_meta.py", line 396, in prepare_metadata_for_build_wheel
    self.run_setup()
[...]
  File "/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools_scm/_integration/setuptools.py", line 102, in version_keyword
    _assign_version(dist, config)
  File "/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools_scm/_integration/setuptools.py", line 54, in _assign_version
    maybe_version = _get_version(config, force_write_version_files=True)
  File "/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools_scm/_get_version_impl.py", line 94, in _get_version
    write_version_files(config, version=version_string, scm_version=parsed_version)
  File "/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools_scm/_get_version_impl.py", line 63, in write_version_files
    dump_version(
  File "/home/moint03/test/moin/.tox/.pkg/lib/python3.9/site-packages/setuptools_scm/_integration/dump_version.py", line 36, in dump_version
    assert not write_to.is_absolute(), f"{write_to=}"
AssertionError: write_to=PosixPath('/home/moint03/test/moin/src/moin/_version.py')

```

--- a/setup.py
+++ b/setup.py
@@ -66,10 +66,10 @@ setup_args = dict(
     include_package_data=True,
     zip_safe=False,
     use_scm_version={
-        'write_to': os.path.join(basedir, 'src', 'moin', '_version.py'),
+        'write_to': os.path.join('src', 'moin', '_version.py'),
     },
     setup_requires=[
-        'setuptools_scm<8.0.0',  # magically cares for version and packaged files
+        'setuptools_scm',  # magically cares for version and packaged files
     ],
     install_requires=[
         'Babel>=2.10.0',  # internationalization support

